src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCase.java

```diff
@@ -54,7 +54,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
         data.getLabelContent().setUnionType(data.getDivorceOrDissolution());
 
         var applicant2 = data.getApplicant2();
-        if (JOINT_APPLICATION.equals(data.getApplicationType())) {
+        if (data.isJudicialSeparationCase()) {
```

Comment:
It seems this if statement does the same as the JOINT_APPLICATION check. Could it be a combined OR statement?

If (isJudicialSeparationCase() || JOINT_APPLICATION.equals(data.getApplicationType()) -> applicant2.setOffline(YES);

Else blah ...

Reply:
I did consider that, and they absolutely could, I just thought it was slightly more obvious/clear this way.